src/cache/messages/data/scalar/get.rs

```diff
@@ -224,22 +227,49 @@ mod tests {
         let hit = GetResponse::Hit {
             value: Value::new("hello".as_bytes().to_vec()),
         };
+        assert_eq!(format!("{}", hit), r#"Value { raw_item: "hello" }"#);
         assert_eq!(
             format!("{:?}", hit),
-            "Hit { value: Value { raw_item: [104, 101, 108, 108, 111] (as string: \"hello\") } }"
+            r#"Hit { value: Value { raw_item: "hello" } }"#
```

Comment:
My only concern here is there could be binary values that can render as utf-8 by happenstance. That said I don't think the raw bytes are all that informative anyway so 🤷